src/instruments/src/EFB/Navigation/Pages/LocalFilesPage/LocalFileChartUI.tsx

```diff
@@ -88,13 +92,14 @@ export const LocalFileChartUI = () => {
         const pdfs: LocalFileChart[] = [];
         const images: LocalFileChart[] = [];
 
+        if (simbridgeEnabled !== 'AUTO ON') {
+            return { images, pdfs }; // No need to search if simbridge is not enabled
+        }
+
         try {
             // IMAGE or BOTH
             if (selectedTabIndex === 0 || selectedTabIndex === 2) {
-                const resp = await fetch('http://localhost:8380/api/v1/utility/image/list');
-
-                const imageNames: string[] = await resp.json();
-
+                const imageNames: string[] = await Viewer.getImageList();
                 imageNames.forEach((imageName) => {
                     if (imageName.toUpperCase().includes(searchQuery)) {
```

Comment:
Have you tested if this produces unwanted behaviour for images that are lower case on purpose?

Reply:
No - this was Erick's code. Need to look into it.

Reply:
Seems ok - it is related to ICAO and searchQuery and the imageName are both transformed to upper case - so the statement makes sense - as windows does not really distinguish between upper and lower I think this is ok.
Also I think this would have caused issues long ago - this is "old" code I did not touch.